pkg/taxonomy/model/base/model_action.go

```diff
@@ -1,133 +0,0 @@
-/*
- * Policy Manager Service
- *
- * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
```

Comment:
@roee88 I had a query. Should the generated code be checked in ? If yes, I think the code will not pass the security checks in the CI as the generated code is written by a third party tool. Please advise. Thanks.

Reply:
I think it must be committed, otherwise code that 
imports it won't compile. For now we can exclude this from go sec (see https://github.com/securego/gosec#excluding-test-files-and-folders) but we need to open an issue for tracking this (e.g., to be fixed by upstream contribution).

Reply:
ok sure. @ritwikc can you please commit the generated code? Thanks. @roee88 regarding the gosec rules, should the changes be done in github actions ? Sorry I am not aware of where to do the gosec related changes.

Reply:
Yes we have a workflow that runs gosec and the chance should be there.

Reply:
Passed gosec with exclude-dir on the generated code directories.